discovery-handlers/onvif/src/discovery_impl.rs

```diff
@@ -323,7 +323,7 @@ pub mod util {
 
             match try_recv_string(socket, time_left).await {
                 Ok(s) => {
-                    broadcast_responses.push(s);
+                    broadcast_responses.insert(s);
```

Comment:
i wonder if we might want to s.ToLower here (i assume that a device would be consistent in how it describes itself, so it seems like this wouldn't be needed ... but something to think about)